### DISCUSSION TITLE
libgpod

### DISCUSSION THREAD

**Discussion by jonaski** [Homebrew/discussions#1396] [Everyday usage]
Hi,

Will you accept a PR to add libgpod. macports have it, but not homebrew.

It is still functional for old iPod's, and we still have users: https://github.com/strawberrymusicplayer/strawberry/issues/703
I also still have a couple of iPods, although I don't use mac.

Worth to note It's not maintained, last release of libgpod was 0.8.3 in 2013. I offered to help out a year ago or so, but no-one in the gtkpod/libgpod project is responding.

It requires libplist, gnome-common, gdk-pixbuf, intltool, autoconf, automake and gtk-doc. All exist in homebrew.
I was able to build it in a macOS VM and in my CI using the following commands:

```
sudo cpan install XML::Parser
wget https://downloads.sourceforge.net/project/gtkpod/libgpod/libgpod-0.8/libgpod-0.8.3.tar.bz2
tar -xvf libgpod-0.8.3.tar.bz2
cd libgpod-0.8.3
wget https://raw.githubusercontent.com/macports/macports-ports/master/multimedia/libgpod/files/autogen.sh
wget https://raw.githubusercontent.com/macports/macports-ports/master/multimedia/libgpod/files/patch-tools-generic-callout.c.diff
wget http://files.strawberrymusicplayer.org/patches/libgpod-libplist.patch
patch -p0 < patch-tools-generic-callout.c.diff
patch -p1 < libgpod-libplist.patch
chmod u+x autogen.sh
./autogen.sh
./configure --disable-more-warnings \
  --disable-silent-rules \
  --disable-udev \
  --disable-pygobject \
  --with-python=no
make -j8
sudo make install
```

**Comment by SMillerDev** [comment]
If it meets https://docs.brew.sh/Acceptable-Formulae you can propose it.
